README.md

```diff
@@ -100,136 +160,80 @@ This will take approximately a minute to run, with the majority of the time spen
+ethereum-node-1-1649105164   ethereum-node-1   8545/tcp -> 127.0.0.1:55907
+                                               8546/tcp -> 127.0.0.1:55908
+                                               30303/tcp -> 127.0.0.1:55909
+                                               30303/udp -> 127.0.0.1:56275
+ethereum-node-2-1649105167   ethereum-node-2   30303/udp -> 127.0.0.1:51099
+                                               8545/tcp -> 127.0.0.1:55964
+                                               8546/tcp -> 127.0.0.1:55965
+                                               30303/tcp -> 127.0.0.1:55963
 
-```javascript
-Set(3) { 'bootnode', 'ethereum-node-1', 'ethereum-node-2' }
 ```
 
 But what just happened?
 
-Starting networks is a very common task in Kurtosis, so we provide [a framework called "modules"](https://docs.kurtosistech.com/modules.html) for making it dead simple. An executable module is basically a chunk of code that responds to an "execute" command, packaged as a Docker image, that runs inside a Kurtosis enclave - sort of like Docker Compose on steroids. In the steps above, we called `enclaveCtx.loadModule` to load [the Ethereum module](https://github.com/kurtosis-tech/ethereum-kurtosis-module) into the enclave with module ID `eth-module`, and `ethModuleCtx.execute` to run it. The Ethereum module doesn't take any parameters at load or execute time (hence the `{}`), but other modules do.
+Starting networks is a very common task in Kurtosis, so we provide [a framework called "modules"](https://docs.kurtosistech.com/modules.html) for making it dead simple. An executable module is basically a chunk of code that responds to an "execute" command, packaged as a Docker image, that runs inside a Kurtosis enclave - sort of like Docker Compose on steroids. In the steps above, we executed `kurtosis module exec` to load [the Ethereum module](https://github.com/kurtosis-tech/ethereum-kurtosis-module) into the demo enclave. The Ethereum module doesn't take any parameters at load or execute time, but other modules do via the `--execute-params` flag.
 
 Now that you have a pet Ethereum network, let's do something with it.
 
 Step Four: Talk To Ethereum (5 minutes)
 ---------------------------------------
-Talking to Ethereum in Javascript is easily accomplished with [the EthersJS library](https://docs.ethers.io/v5/). Your Javascript REPL is running in a Docker image (so that you don't need Javascript installed locally), so we'll need to install EthersJS on that image.
 
-First, in a new terminal window, run the following to find the enclave our REPL is running inside:
+Now let's connect to the Ethereum bootnode to verify the network is producing blocks.
 
-```
-kurtosis enclave ls
-```
+Find the Ethereum node with ID bootnode in the enclave contents, find its RPC port declared on 8545/tcp, and copy the public IP and port that it's bound to on your machine (e.g. 127.0.0.1:55903).
```

Comment:
Can we wrap the following in backticks:
bootnode -> `bootnode`
8545/tcp
127.0.0.1:55903

? I want to emphasize to users that these are things in their terminal